src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css

```
@@ -2,10 +2,14 @@
   display: grid;
   grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
   grid-template-rows: 465px;
-  margin: auto;
+  margin: auto auto 42px;
```

I think we don't actually even really want the `auto` on the top side, right? Maybe this should be:

```suggestion
  margin: 0 auto 42px;
```

---

ok